NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 14.1.2 2020-04-09
+
+* Fix processing of job profiles in json format by MatchEngine (MODINV-271)
+* Fix matching on Instance fields (MODINV-275)
+* Fix Items creation upon receiving an event (MODINV-278)
```

Comment:
I'm aware I've already provided feedback on these points already, and so this might be redundant.

Is it possible to be more specific about what was fixed? For example, under what circumstances did processing of job profiles fail, or which instance fields failed to match?